.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: HACF Discord
+    url: https://discord.gg/PaZFEjX
+    about: The HACF discord server
```

Comment:
Thank you for your contribution. To motivate user to use Discord and Foru, can you put details in the `about` key explaining why they should use the discord. It could be good idea to specify the Discord is french speaking.